presto-native-execution/presto_cpp/main/tests/TaskManagerTest.cpp

```diff
@@ -823,4 +830,59 @@ TEST_F(TaskManagerTest, aggregationSpill) {
   }
 }
 
+TEST_F(TaskManagerTest, createOrUpdateTask) {
```

Comment:
Test name is quite generic, but I believe this test is verifying a particular scenario. Would you document that scenario and see if there is a more specific test name?